staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/policy_decision.go

```diff
@@ -47,7 +47,7 @@ type PolicyDecision struct {
 	Elapsed    time.Duration
 }
 
-type policyDecisionWithMetadata struct {
+type PolicyDecisionWithMetadata struct {
```

Comment:
+godoc on exported type?

Reply:
done, and moved to validatingadmissionpolicy package since its specific to that package